src/site/content/en/blog/wakelock/index.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Stay awake with the WakeLock API
+subhead: The Wake Lock API provides a way to prevent the device from dimming or locking the screen when an application needs to keep running.
+authors:
+  - petelepage
+  - thomassteiner
+description: To avoid draining the battery, most devices will quickly fall asleep when left idle. While this is fine most of the time, there are some applications that need to keep the screen or the device awake in order to complete some work. The Wake Lock API provides a way to prevent the device from dimming or locking the screen or prevent the device from going to sleep when an application needs to keep running.
+date: 2018-12-18
+updated: 2019-10-17
+tags:
+  - post
+  - capabilities
+  - wake-lock
+hero: hero.jpg
+alt: Sleeping koala
+draft: true
+---
+
+{% Aside %}
+  The Wake Lock API, is part of
+  [capabilities project](https://developers.google.com/web/updates/capabilities),
+  and is currently in development, this post will be updated as the
+  implementation progresses.
+{% endAside %}
+
+## What is the Wake Lock API? {: #what }
+
+To avoid draining the battery, most devices quickly go to sleep when left
+idle. While this is fine most of the time, some applications need to keep the
+screen or the device awake in order to complete their work. Examples include a
+run-tracking app (turns the screen off, but keeps the system awake), or a game,
+like [Ball Puzzle](https://ball-puzzle.appspot.com/), that uses the device
+motion APIs for input.
+
+The [Wake Lock API][spec-ed] provides a way to prevent the device from dimming
+and locking the screen or to prevent the device from going to sleep. This
+capability enables new experiences that, until now, required a native app.
+
+The Wake Lock API reduces the need for hacky and potentially
+power-hungry workarounds. It addresses the shortcomings of an older API
+which was limited to simply keeping the screen on, and had a number of
+security and privacy issues.
+
+### Suggested use cases for the Wake Lock API {: #use-cases }
```

Comment:
I see your point, what I liked about this example is that it is a concrete example. Since the API isn't really available yet, everything else is essentially hypothetical.

Reply:
Fair enough!